plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-products-catalog-controller.php

```
@@ -0,0 +1,244 @@
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'generate_products_catalog',
+			'type'       => 'object',
+			'properties' => array(
+				'job_id' => array(
+					'description' => __( 'Products catalog generation job ID.', 'woocommerce' ),
+					'type'        => 'string',
+					'required'    => true,
+				),
+			),
+		);
+	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'catalog_generation_schema'. (undefined)

(UnusedPrivateMethod)